todo/src/components/TodoList.js

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from 'react';
+import { connect } from "react-redux";
+import { fetchTodos , deleteTodo } from "../actions";
+
+class TodoList extends Component {
+
+    componentDidMount() {
+        this.props.fetchTodos();
+    }
+
+    toggleComplete = (e) => {
+        if (e.target.style.textDecoration === 'line-through') e.target.style.textDecoration = 'none';
+        else e.target.style.textDecoration = 'line-through';
```

Comment:
This is an efficient way of handling this, but there's something you should keep in mind: If you want this part of the data (whether it's "complete" or not) to persist across sessions, you'll need to set it as a flag on the todo object that you're passing to your `addToDo` action.